SPECS-EXTENDED/doxygen/doxygen.spec

```diff
@@ -1,13 +1,13 @@
 Name:           doxygen
 Version:        1.8.17
-Release:        2%{?dist}
+Release:        3%{?dist}
 Summary:        Automated C, C++, and Java Documentation Generator
 License:        GPL
```

Comment:
Please use one of the short license names from [this table](https://fedoraproject.org/wiki/Licensing:Main?rd=Licensing#Good_Licenses).

Reply:
Can you help what should be added?

Reply:
Sure: take a look at the contents of the `LICENSE` file in the sources (the one from the `%license` macro). Judging from the current `License` tag, it's probably going to be some version of GPL. If it's 2.0 or 2.1, then you'd write `GPLv2+`. If it's 3.0 or higher, then `GPLv3+`. If by any chance the license is completely different, then, first of all, it's good that we checked, and you'd just need to find it in the table and use the right name.

Reply:
Updated